includes/gateway/class-omise-payment-creditcard.php

```diff
@@ -555,8 +515,8 @@ public function callback() {
 			try {
 				$charge = OmiseCharge::retrieve( $this->get_charge_id_from_order() );
 
-				switch ( strtoupper( $this->payment_action ) ) {
-					case 'MANUAL_CAPTURE':
+				switch ( $this->payment_action ) {
```

Comment:
`strtoupper` has been removed - is this because it wasn't actually needed? Or is this a potentially dangerous mistake?

Reply:
@jonrandy  It wasn't actually needed (I was afraid if it would raise any concerns so I put it there to make sure but in fact, this value has been set by its constant's value anyway, so it should always be a lower-case)